scripts/test.sh

```diff
@@ -50,23 +50,22 @@ function ensure_trailing_whitespace_free() {
 }
 
 function bootstrap_examples() {
-  for file in ${EXAMPLES_DIR}/*; do
-    if [ -d "${file}" ]; then
-      bootstrap_script="${file}/bootstrap.sh"
-      if [ -x "${bootstrap_script}" ]; then
-        echo "    Bootstrapping ${file}..."
+  while read -r file;
+  do
+    bootstrap_script="${file}/bootstrap.sh"
+    if [ -x "${bootstrap_script}" ]; then
+      echo "    Bootstrapping ${file}..."
 
-        env_vars=$(${bootstrap_script})
-        bootstrap_result=$?
-        if [ $bootstrap_result -ne 0 ]; then
-          echo "    Running bootstrap script (${bootstrap_script}) failed!"
-          exit 1
-        else
-          eval $env_vars
-        fi
+      env_vars=$(bash "${bootstrap_script}")
```

Comment:
Can we do this another way? It's unfortunate to have to call `bash` directly here.